.azure-pipelines-templates/deploy_attestation_container.yml

```diff
@@ -0,0 +1,103 @@
+jobs:
```

Comment:
This reuses some parts of `deploy_aci.yml`. Would it be possible to refactor this to have a single consistent way to deploy ACI containers?

Reply:
Could we have some minimal reusable YAML building blocks (e.g. install the azure CLI) that we can call from both pipelines, possibly with different arguments so that we can easily compose (and read!) pipelines that deploy and/or test ACI containers. We follow this pattern already in some part of the CI, e.g. [here](https://github.com/microsoft/CCF/blob/main/.azure-pipelines-templates/common.yml#L51-L70). This can of course be done in a separate refactoring PR!

Reply:
Let's do this separately, after this PR.